docs/products/clickhouse/concepts/features-overview.rst

```diff
@@ -0,0 +1,106 @@
+Aiven for ClickHouse® features overview
+=======================================
+
+What is Aiven for ClickHouse® and what are its key features? Why and when to use Aiven for ClickHouse? What are its most common use cases? This article helps you discover it all and more.
+
+About Aiven for ClickHouse
+--------------------------
+
+Aiven for ClickHouse® is a fully managed distributed columnar database which is built on the open source ClickHouse datastore. It is a fast highly-scalable fault-tolerant solution designed for online analytical processing and data warehousing. Aiven for ClickHouse enables you to execute complex SQL queries on large datasets quickly and effectively to process large amounts of data in real time.
```

Comment:
I think we should use a more powerful word than datastore. Datastore is kind of limiting the meaning towards the datawarehouse use-case but ClickHouse is also very powerful on the online analytical processing (OLAP) use-case which we should not diminish. A generic term solution could work here IMHO.